src/include/apparmor/aa-config.rb

```diff
@@ -33,20 +33,26 @@ def initialize_apparmor_aa_config(include_target)
       Yast.import "Label"
       Yast.import "Popup"
       Yast.import "Wizard"
+      Yast.import "Service"
     end
 
     def changeAppArmorState(aaEnabled)
       error = nil
       aaAction = ""
```

Comment:
both variables no longer needed, as error is written later and aaAction is not used anymore.